src/files/coverageparser.ts

```diff
@@ -51,7 +51,7 @@ export class CoverageParser {
                     coverage = await this.xmlExtractCobertura(fileContent);
                     break;
                 case CoverageType.LCOV:
-                    coverage = await this.lcovExtract(fileContent);
+                    coverage = await this.lcovExtract(file[0], fileContent);
```

Comment:
to prep for me applying a similar to fix the other parsers, this (`file[0]`) should be assigned to a new variable near to line 34 where `fileContent` is.